src/app/components/actions/actions.tsx

```diff
@@ -1,14 +1,19 @@
+import { ClickableText } from '../clickable-text';
+import { Description } from '@/features/required/required.types';
+
 import './actions.scss';
 
 interface ActionsProps {
-  actions: string[];
+  actions: Description;
   marked?: boolean;
 }
 export const Actions = ({ actions, marked = false }: ActionsProps) => {
   return (
     <ul className={`stage-actions ${marked ? 'marked' : ''}`}>
       {actions.map((action, index) => (
-        <li key={index}>{action}</li>
+        <li className={`${typeof action !== 'string' ? 'marked' : ''}`} key={index}>
+          {typeof action === 'string' ? <>{action}</> : <ClickableText data={action} />}
+        </li>
       ))}
     </ul>
   );
```

Comment:
Is not it a bad practice to use index as a `key` property? Shouldn't we use some other value like link (as I see links are unique)?

Reply:
In this current example the `key` is consistent across renders, because the order of the elements in the array will never change, therefore no performance drop will be produced.

But you totally right that the index as a `key` should be avoided.

There is a chance that the links can intersect, i would prefer to add `id` to each item in the array 

![image](https://github.com/rolling-scopes/site/assets/114234698/6bd6be22-2d96-4332-a39d-ef7008685161)

Reply:
This item has been fixed.

Reply:
I belive you can replace string interpolation like this:

```tsx
return (
    <ul className="stage-actions">
      {actions.map((action) => {
        const isLink = typeof action !== 'string';
        const itemClassName = isLink || marked ? 'marked' : undefined;
        const keyAction = isLink ? action[0].id : action;

        return (
          <li className={itemClassName} key={keyAction}>
            {isLink ? <ClickableText data={action} /> : action}
          </li>
        );
      })}
    </ul>
  );
};
```

This will produce
```tsx
const itemClassName = isLink || marked ? 'marked' : undefined;
<li className={itemClassName} key={keyAction}>
 //...
</li>
```

the same result as this
```tsx 
className={`stage-actions ${marked ? 'marked' : ''}`
```